File: irrgarten-java/src/tests/TestP3.java
package tests;

import irrgarten.Game;
import irrgarten.ui.TextUI;
import irrgarten.controller.Controller;


public class TestP3 {
    public static void main(String[] args) {
        Game g = new Game(1);
        TextUI i = new TextUI();
        Controller c = new Controller(g, i);
        c.play();
    }
}

File: irrgarten-ruby/test_p3.rb
# encoding: utf-8

require_relative 'player'
require_relative 'labyrinth'
require_relative 'game'
require_relative 'text_ui'
require_relative 'controller'

module Irrgarten

  class TestP3
    include UI
    include Control

    def self.main
      p = Player.new(1,2,3)
      l = Labyrinth.new(10,10,5,6)

      l.spread_players([p])
      l.put_player(Directions::DOWN, p)

      puts l.to_s
      puts p.to_s

      g = Game.new(2)
      ui = TextUI.new
      con = Controller.new(g,ui)
      con.play
    end
  end

end


Irrgarten::TestP3.main